test/data/vue-loader/components/app.vue

```diff
@@ -0,0 +1,16 @@
+<template lang="pug">
```

Comment:
` test/data/index.html → test/data/simple-page/index.html`

Let's rename to `test/data/vuejs/index.html`

Reply:
may be rename it as `vue-js`
I think it will look better
`data`
`- vue-js`
`- vue-loader`